fastlane_bot/modes/triangle_multi.py

```diff
@@ -8,79 +8,72 @@
 All rights reserved.
 Licensed under MIT.
 """
-from typing import List, Any, Tuple, Union
+from typing import Any, List
+from itertools import product
 
 from fastlane_bot.modes.base_triangle import ArbitrageFinderTriangleBase
-from fastlane_bot.tools.cpc import CPCContainer
-from fastlane_bot.tools.optimizer import MargPOptimizer
-
 
 class ArbitrageFinderTriangleMulti(ArbitrageFinderTriangleBase):
-    """
-    Triangular arbitrage finder mode
-    """
+    def get_combos(self) -> List[Any]:
+        combos = []
+        all_carbon_curves = self.CCm.byparams(exchange="carbon_v1").curves
+
+        for flt in self.flashloan_tokens:
+            non_flt_carbon_curves = [curve for curve in all_carbon_curves if flt not in curve.pair]
+            for non_flt_carbon_curve in non_flt_carbon_curves:
+                tkny = non_flt_carbon_curve.tkny
+                tknx = non_flt_carbon_curve.tknx
+
+                carbon_curves = self.CCm.bypairs(f"{tknx}/{tkny}").byparams(exchange="carbon_v1").curves
+                if len(carbon_curves) == 0:
```

Comment:
The check for carbon curves here should be removed - there is no need to fail this mode if no Carbon Curves exist.